It's ridiculous to add a screenful of legal guacamole to a one-liner
patch, but it seems some weenies won't do without.

This patch removes the 8BITMIME advertisement from qmail-smtpd's EHLO
reply, because qmail does not implement 8BITMIME properly. If you feel
this is not correct, make a better patch. You have two choices: a) tell
qmail about quoted-printable encoding and MIME (not recommended), b) let
qmail bounce mail with 8bit-data that is shipped to 7bit destinations.
(Will probably require queue modifications.)

Copyright (c) 2002 Matthias Andree

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in
all copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
THE SOFTWARE.

*** ./qmail-smtpd.c.orig	Mon Jun 15 12:53:16 1998
--- ./qmail-smtpd.c	Tue May 28 19:13:12 2002
***************
*** 229,235 ****
  }
  void smtp_ehlo(arg) char *arg;
  {
!   smtp_greet("250-"); out("\r\n250-PIPELINING\r\n250 8BITMIME\r\n");
    seenmail = 0; dohelo(arg);
  }
  void smtp_rset()
--- 229,235 ----
  }
  void smtp_ehlo(arg) char *arg;
  {
!   smtp_greet("250-"); out("\r\n250 PIPELINING\r\n");
    seenmail = 0; dohelo(arg);
  }
  void smtp_rset()
